docs/installation.md

```diff
@@ -81,6 +81,14 @@ $ npm start
 
 7. open `index.html` in a browser
 
+Optionally, you can define mithril as a webpack plugin, the benefit is that mithril will be included globally and you wont have have to `import m from "mithril"`:
+```json
+plugins: [
+    new webpack.ProvidePlugin({m: 'mithril'}),
+    // ...
+    ]
+```
+Then you could remove the import line from step 4 (dont forget to restart webpack if you ran it with `--watch`) and it will work just the same
```

Comment:
Please add a newline after this for consistency with everything else.

Reply:
Done, github doesnt show it in the main Conversations because it takes it as part of the following block for some reason but if you check [today's commit](https://github.com/MithrilJS/mithril.js/pull/2608/commits/740620c1fe451bd113a6e8056316eb171041391c), its there